hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SecurityUtil.java

```diff
@@ -106,6 +110,9 @@ private static void setConfigurationInternal(Configuration conf) {
     boolean useIp = conf.getBoolean(
         CommonConfigurationKeys.HADOOP_SECURITY_TOKEN_SERVICE_USE_IP,
         CommonConfigurationKeys.HADOOP_SECURITY_TOKEN_SERVICE_USE_IP_DEFAULT);
+    cachingInterval = conf.getInt(
+        CommonConfigurationKeys.HADOOP_SECURITY_HOSTNAME_CACHE_EXPIRE_INTERVAL_SECONDS,
+        CommonConfigurationKeys.HADOOP_SECURITY_HOSTNAME_CACHE_EXPIRE_INTERVAL_SECONDS_DEFAULT);
```

Comment:
If this is a timeout duration, we recommend using `conf.getTimeDuration(...)` to retrieve it. We generally do not use the `getInt(...)` approach.

Reply:
https://github.com/apache/hadoop/blob/cc9e54857278062b8f46233bbcfbc9dd975507cf/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java#L3717-L3721

Reply:
Good Point!
I agree that getTimeDuration() is more suitable here.
I'll make this change soon.